File: Hydrogene/Nougat/UIKit/UITableViewCell/UITableViewCell/OlympicCityTableViewCell.cs
using UIKit;

namespace CSharpiOSTableViewCellDemo
{
    [IBObject]
    public class OlympicCityTableViewCell : UITableViewCell
    {
        [IBOutlet]
        public UILabel city;
        [IBOutlet]
        public UILabel country;
        [IBOutlet]
        public UILabel gamesType;
        [IBOutlet]
        public UILabel year;
        [IBOutlet]
        public UILabel notes;

    }
}

File: Oxygene/Toffee/UIKit/UITableViewCell/UITableViewCell/OlympicCityTableViewCell.pas
namespace OxygeneiOSUITableViewCellDemo;

interface

uses
  UIKit;

type
  [IBObject]
  OlympicCityTableViewCell = public class(UITableViewCell )
  private
  public
    [IBOutlet] city : UILabel;
    [IBOutlet] country : UILabel;
    [IBOutlet] gamesType : UILabel;
    [IBOutlet] year : UILabel;
    [IBOutlet] notes : UILabel;

    method awakeFromNib; override;
  end;

implementation

method OlympicCityTableViewCell.awakeFromNib;
begin
  inherited;
  NSLog("Cell Created");
end;



end.
